coverage/index.html

```diff
@@ -64,7 +64,7 @@ <h1>
 </div><!-- /wrapper -->
```

Comment:
Please make sure `/coverage/*` is in your `.gitignore` file and run `git rm coverage/index.html` to remove this file from your git history